This is a bogus dep

--- origsrc/kactivities-5.27.0/src/imports/CMakeLists.txt	2016-10-02 02:44:54.000000000 -0500
+++ src/kactivities-5.27.0/src/imports/CMakeLists.txt	2016-11-02 09:20:37.913810600 -0500
@@ -7,7 +7,6 @@ set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PA
 find_package (Qt5 REQUIRED NO_MODULE COMPONENTS Sql Gui Qml Quick Sql)
 find_package (KF5Config     ${KF5_DEP_VERSION} CONFIG REQUIRED)
 find_package (KF5CoreAddons ${KF5_DEP_VERSION} CONFIG REQUIRED)
-find_package (KF5KIO        ${KF5_DEP_VERSION} CONFIG REQUIRED)
 
 set (
    kactivities_imports_LIB_SRCS
@@ -33,7 +32,6 @@ target_link_libraries (
    KF5::Activities
    KF5::ConfigCore
    KF5::CoreAddons
-   KF5::KIOCore
    )
 
 ## install
